### ISSUE TITLE
ExceptionSerializerFactory Broken on DotNet Core 3 Preview 4

### ISSUE THREAD

**Issue by netclectic** [akkadotnet/Hyperion#111]
Experimenting with DotNet Core 3, the recent update for preview 4 breaks the ExceptionSerializerFactory.

Specifically this:
```
        // Workaround for CoreCLR where FormatterServices.GetUninitializedObject is not public
        private static readonly Func<Type, object> GetUninitializedObject =
            (Func<Type, object>)
                typeof(string).GetTypeInfo().Assembly.GetType("System.Runtime.Serialization.FormatterServices")
                    .GetMethod("GetUninitializedObject", BindingFlags.NonPublic | BindingFlags.Public | BindingFlags.Static)
                    .CreateDelegate(typeof(Func<Type, object>));
```
causes a null reference exception complaining that the type parameter cannot be null.

It seems the FormatterServices have been moved. I was able to resolve it by making a local NetStandard2 specific build and changing to use typeof directly on System.Runtime.Serialization.FormatterServices.

**Comment by heatonmatthew** [comment]
Yes I found the same.  I've got a patch that adds netstandard2.0 to the build targets and then compiles conditionally for the target framework which seems to work.

**Comment by Alois-xx** [comment]
I also found it. When will be the next release pushed? If it does not work I will have to remove it from my SerialierTest suite for .NET Core 3.0.  

https://github.com/Alois-xx/SerializerTests

**Comment by OnurGumus** [comment]
any love for this issue? It is really a show stopper for .net core 3 apps.

**Comment by Danthar** [comment]
Fixed with #119
